roles/docker_rmi/docker_rmi_httpd_image/tasks/main.yml

```diff
@@ -0,0 +1,32 @@
+---
+# vim: set ft=ansible:
+#
+- name: Fail if drmi_image_name is not defined
+  when: drmi_image_name is not defined
+  fail:
+    msg: "The variable drmi_image_name must be defined"
+
+- name: Get all the docker images
+  command: docker images
+  register: images_before
+
+- name: Fail if the image is not present
+  when: drmi_image_name not in images_before.stdout
+  fail:
+    msg: |
+      Expected: {{ drmi_image_name}} container is in docker images
+      Actual: docker images {{ images_before.stdout }}
+
+- name: Remove the image
+  command: "docker rmi -f {{ drmi_image_name }}"
+
+- name: Get the docker images against
+  command: docker images
+  register: images_after
+
+- name: Fail if the image is present
+  when: drmi_image_name in images_after.stdout
+  fail:
+    msg: |
+      Expected: {{ drmi_image_name }} is removed from docker images
+      Actual: docker images: {{ images_after.stdout }}
```

Comment:
Is this file/directory misplaced?
`roles/docker_rmi/docker_rmi_httpd_image/tasks/main.yml` should just be `roles/docker_rmi/tasks/main.yml`

Same deal with the meta dir and main.yml file for docker_rmi.

Reply:
Or were these supposed to be deleted?

Reply:
Crap.  Probably got things messed up when I rebased.  I'll sort it out.